Week2/week1/5_variable-z.js

```diff
@@ -0,0 +1,22 @@
+let z = 7.25;
+
+console.log(z);
+
+let a = Math.round(z);
+
+console.log(a);
+
+let highest;
+
+if(a > z)

```

Comment:
Nice solution! You can also use [math.max](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max). But this is a fine solution!